### PR TITLE
new strings: for Slack posts

### PR DESCRIPTION
**Pull request by mab253**
points to SMS map instead of suggesting that users comment on a Slack thread to pick up delivery!

[also adding new Twilio audio file, from serverless update around x-mas time]

### CODE REVIEW

**Comment by mab253** [comment]
[please ignore how that last commit is called "major update" hahah - that was for my own forked repo!]

**Review by piratefsh — APPROVED** [review]
one question but otherwise lgtm!!

**Comment by mab253** [comment]
> one question but otherwise lgtm!!

cool! wait i'm not seeing the question anywhere @piratefsh - happy to answer tho!

File: src/twilio/functions/sms/intake-delivery-txts.js [hunk 1]
```diff
@@ -20,7 +20,6 @@ exports.handler = function codeSms(context, event, callback) { // eslint-disable
   // below: looks up code in request Airtable, gets all important values about the requesting neighbor
   base("Requests")
     .select({
-      view: "Grid view", // from staging base
```
Comment (piratefsh) [inline src/twilio/functions/sms/intake-delivery-txts.js:23]: @mab253 oh sorry my Q was this change intentional? just checking since this wasnt mentioned in the desc!

Comment (mab253) [reply src/twilio/functions/sms/intake-delivery-txts.js:23]: o yes! i believe this is already deployed via the twilio flow - but this is because in the "Requests" view that's not the Staging base, there's no view called "Grid view." And if we don't add the view parameter, it searches through the whole table 👍 

thnx for flagging!!
